packages/ppx/test/native/At_rule_test.re

```diff
@@ -83,25 +83,20 @@ let media_query_tests = [
     [%expr CSS.style([|CSS.media({js|screen, print|js}, [||])|])],
   ),
   (
-    "(screen, print)",
-    [%expr [%cx "@media (screen, print) {}"]],
-    [%expr CSS.style([|CSS.media({js|(screen, print)|js}, [||])|])],
-  ),
-  (
-    "(screen and (color))",
-    [%expr [%cx "@media (screen and (color)) {}"]],
-    [%expr CSS.style([|CSS.media({js|(screen and (color))|js}, [||])|])],
+    "((screen) and (color))",
+    [%expr [%cx "@media ((screen) and (color)) {}"]],
```

Comment:
This looks incorrect, the right way would be 

```
@media screen and (color) {
  color: blue;  
}
```

Parsing is: `@media [media-type] and ([media-feature]) { }`

Reply:
i think we tested that already, this should resolve to this https://csstree.github.io/docs/syntax/#Type:media-query-list
![wat](https://github.com/user-attachments/assets/bd755d5e-8bd9-4154-92ae-bc3768c36c13)